utils/src/main/java/de/ergodirekt/drag/utils/Copy.java

```diff
@@ -1,14 +1,20 @@
-import java.io.*;
+package de.ergodirekt.drag.utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
 
 public class Copy {
-    public static void main(String[] args) {
+
+    public static void main(String[] args) throws DragException {
         File srcFolder = new File("C:\\Users\\Administrator\\Desktop\\hello");
-        File destFolder = new File("C:\\Users\\Administrator\\Documents\\newHello\\");
+        File destFolder = new File("C:\\Users\\Administrator\\Documents\\newHello");
 
         if (!srcFolder.exists()) {
-            System.out.println("Directory does not exist.");
-            System.out.println("Good bye1 ");
-            System.exit(1);
+                throw new DragException("Directory does not exist.");
```

Comment:
Die Überprüfung der Verzeichnisse würde ich in der Service-Methode machen.
Biete eine Einstiegsmethode mit zwei Strings für src und dest an.